src/StrokeEngine.cpp

```diff
@@ -746,86 +247,20 @@ void StrokeEngine::_stroking() {
         }
         
         // Delay 10ms 
-        vTaskDelay(10 / portTICK_PERIOD_MS);
-    }
-}
-
-void StrokeEngine::_streaming() {
-
-    while(1) { // infinite loop
-
-        // Suspend task, if not in STREAMING state
-        if (_state != STREAMING) {
-            vTaskSuspend(_taskStreamingHandle);
-        }
-        
-        // Delay 10ms 
-        vTaskDelay(10 / portTICK_PERIOD_MS);
-    }
-}
-
-void StrokeEngine::_applyMotionProfile(motionParameter* motion) {
-
-    bool clipping = false;
-    float speed = 0.0;
-    float position = 0.0;
-
-    // Apply new trapezoidal motion profile to servo if pattern does not skip
-    if (motion->skip == false) {

```

Comment:
Small little details easily missed. This skipping feature allows a pattern to insert pauses with no motion. We should find a solution for this.

Reply:
Yeah, I almost wanted to allow Patterns to send two different types of Motion Commands. Move and Pause. I had that in the initial idea, but it never made it into implementation. I will have to think about how to best handle this.

I'm partially thinking the Patterns should output T-Codes, and the T-Code hopefully has a pause for time / condition code. I'd really have to think on it more to see if that's appropriate though

Reply:
I can definitely also just put skipping back in for this first prototype, especially as I definitely un-intentionally removed it

Reply:
I checked, and skipping is still in the new _stroking function. It should still work as well. Resolving comment